.github/workflows/test.yml

```diff
@@ -223,7 +223,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install "mysqlclient>=1.4,<2"
+          pip install mysqlclient
```

Comment:
The ability to set the charset on connect was added in 2.2.0 as per https://github.com/wagtail/wagtail/issues/9477#issuecomment-1823599286.